pkg/services/authn/authnimpl/hook/default_org_hook.go

```diff
@@ -0,0 +1,74 @@
+package hook
```

Comment:
All other hooks live in "sync" package. We should probably put this one there. Also we could probably just add this as a function on `OrgSync`.

Reply:
First I put it into the hook package, because it's a bit different like the other hooks in the a "sync" package (not executed based on the ClientParams) like most of the others, but I don't have a strong opinion on this so I moved it to the "sync" package.

Reply:
Yeah I think you can even move the Hook function itself to OrgSync structure. They share the same dependecies, no need to have a separate struct for it